doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1851,7 +1939,7 @@ NCSDK-18112: :ref:`bluetooth_central_dfu_smp` sample cannot do discovery on the
 
```

Comment:
Please add a new issue (no workaround available yet):
```
NCSDK-26424: Directed advertising in :ref:`peripheral_hids_mouse` does not start after disconnecting from a bonded peer.
When the sample disconnects from a peer, after successful pairing and subscription to reports, it cannot re-connect because directed advertising does not start.
```

Reply:
Done